gnucash/gnome-utils/gnc-main-window.cpp

```diff
@@ -3642,61 +3837,334 @@ gnc_main_window_get_action_group (GncMainWindow *window,
     if (entry == nullptr)
         return nullptr;
 
-    return entry->action_group;
+    return entry->simple_action_group;
```

Comment:
Same remark regarding merged_actions_table. I think this is what is needed here:
```
return gtk_widget_get_action_group (GTK_WIDGET(window), group_name);
```

All but the early sanity tests (g_return_val_if_fail) is no longer needed.